public/data/shivansh0203.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Shivansh Singh",
+  "location": "Lucknow, India",
+  "bio": "An aspiring web developer who is an undergrad and recently started exploring open source. Looking for some great opportunities in fullstack development.",
+  "avatar": "https://github.com/shivanshsin0202.png",
```

```suggestion
  "avatar": "https://github.com/shivanshsin0203.png",
```
Correct your username in avatar line and also resolve merge conflicts.